Automatic Validation ended with:
> 2024-01-25T03:05:20.8663489Z ##[error] Manifest Error: Invalid field value. [InstallerType]

(Automated response - build 655.)

--- a/manifests/g/Google/AndroidStudio/2022.3.1.21/Google.AndroidStudio.installer.yaml
+++ b/manifests/g/Google/AndroidStudio/2022.3.1.21/Google.AndroidStudio.installer.yaml
@@ -3,6 +3,7 @@
 
 PackageIdentifier: Google.AndroidStudio
 PackageVersion: 2022.3.1.21
+InstallerType: nullsoft
 Scope: machine
 InstallModes:
 - interactive

--- a/manifests/g/Google/AndroidStudio/2022.3.1.21/Google.AndroidStudio.installer.yaml
+++ b/manifests/g/Google/AndroidStudio/2022.3.1.21/Google.AndroidStudio.installer.yaml
@@ -11,7 +11,8 @@ InstallModes:
 InstallerSuccessCodes:
 - 1223
 UpgradeBehavior: install
-ProductCode: Android Studio
+AppsAndFeaturesEntries:
+- ProductCode: Android Studio
 Installers:
 - Architecture: x64
   InstallerUrl: https://redirector.gvt1.com/edgedl/android/studio/install/2022.3.1.21/android-studio-2022.3.1.21-windows.exe

